_pages/platform/pipeline.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: article
+title:  "ML Pipelines"
+excerpt: "Creating ML pipelines with Algorithmia."
+categories: basics
+tags: [basics]
+show_related: true
+image:
+    teaser: /icons/algo.svg
+permalink: /platform/pipeline/
+---
+
+# ML Pipelining in Algorithmia
+
+Algorithmia was built from the ground up with machine learning at scale use cases in mind. Pipelining is a key part of any full scale deployment solution. Teams need to be able to productionize models as parts of a whole. This means two things: first, splitting up your machine learning workflows into independent, reusable, modular parts that can then be pipelined together to create models. This makes building models more efficient, cutting out redundant work. Second, we want to automate the machine learning workflow by enabling data to be transformed and correlated into a model that can then be analyzed to achieve outputs, streamlining the process of inputting data into the ML model.
+
+A pipelining architecture solves the problems that arise at scale:
+
+- Volume: only call parts of the workflow when you need them, and cache or store results that you plan on reusing.
+- Variety: when you expand your model portfolio, you can use pieces of the beginning stages of the workflow by simply pipelining them into the new models without replicating them.
+- Versioning: when services are stored in a central location and pipelined together into various models, there is only one copy of each piece to update. All instances of that code will update when you update the original.
```

Comment:
```suggestion
- Versioning: when services are stored in a central location and pipelined together with various models, there's only one copy of each piece of the pipeline to update. Because services in multiple different pipelines may be calling the same upstream service, when the original upstream services is updated, all downstream services will have access to that updated version immediately, and code doesn't need to be changed in multiple places.
```